_data/banking.yml

```diff
@@ -1,4 +1,4 @@
-websites:
+﻿websites:
```

Comment:
Please undo this change.

Reply:
I believe this may have happened when I accidentally hit delete on the top line when editting, and corrected it, by undoing.  I did a diff and they look identical currently compared to the original.  Also it appears the checks have passed as well, and I double checked other pull requests to make sure the spacing, line placement, etc. is correct.  Am I missing something?

Reply:
If you edit this file directly on github's website, you'll see there's a character in front of `websites`. I can't tell what exactly it is, but that is what causing this diff. Please delete that char.

Reply:
I see it.  Looks like an extra space, compared to the original.  It has been removed